feat(Auto Layout): Make Padding to behave like figma Auto Layout (#375)
unoplatform/uno.toolkit.ui: doc/controls/AutoLayoutControl.md

Children |  AutoLayoutChildren | List of FrameworkElement that are the AutoLayout children
PrimaryAxisAlignment | AutoLayoutAlignment | Indicates where the content be displayed on the axis of the AutoLayout orientation
IsReverseZIndex | bool | Reverse the Z-index of all the children

> Padding | Thickness | /!\ Auto Layout padding behave like in the figma pluging : The anchors points determine for which sides the padding is taken in consideration. Exemple if the items are align in Right and Top position, only the Right and Top padding value will affect them.   

Reviewer: ```suggestion
Padding | Thickness | /!\ Padding for `AutoLayout` should behave the same as it does within the Figma Plugin: The anchor points determine which sides of the Padding will taken into consideration. For example, items that are aligned to the Right and Top positions will only take the `Tickness.Right` and `Thickness.Top` values of `Padding` into consideration.
```

I cleaned up the grammar here but to be honest I have absolutely no idea what this property is doing. Are we saying that applying a Padding to the AutoLayout control won't behave the same as it would when setting Padding on any other type of control? 

Perhaps we need a better property here, like `PaddingMask` with values of Top, Left Right, Bottom?

Reviewer 2: No, this control mimics figma layout behavior. Given that this is a _temporarily_ control, it is fine to leave it as close as possible to figma, both in properties and in behaviors.
Btw, IIRC it will eventually be replaced by Grid and appropriate properties (in uwp terms).

Author: thanks !

CounterAlignment|AutoLayoutAlignment|Indicates where an element should be displayed on the counter axis of the parent orientation relative to the allocated layout slot of the parent element
PrimaryLength|double| Set the grid size in parent orientation [Deprecated should Height and Width instead] 
CounterLength|double|Sets the child size in parent counter orientation [Deprecated should Height and Width instead] 

> IsIndependentLayout  | bool | Make the child independent of the AutoLayout positioning. Should not be used with other Attached Properties. Reflect the Absolute Position option from Figma.

Reviewer: We should either commit to ending each property description with a period or none of them, right now it's kind of all mixed

Reviewer 2: and in addition, unify the spacing too: (pick one) space around `|` pipe OR no space around it.
the very least no double spacing:
```suggestion
IsIndependentLayout | bool | Make the child independent of the AutoLayout positioning. Should not be used with other Attached Properties. Reflect the Absolute Position option from Figma.
```

Author: thanks !
